src/System.Management.Automation/CoreCLR/CorePsAssemblyLoadContext.cs

```diff
@@ -46,7 +41,16 @@ internal static PowerShellAssemblyLoadContext InitializeSingleton(string basePat
             lock (s_syncObj)
             {
                 if (Instance != null)
+                {
                     throw new InvalidOperationException(SingletonAlreadyInitialized);
+                }
+
+                // Add last resort native dll resolver.
+                // Default order:
+                //      1. AssemblyLoadContext.LoadUnmanagedDll()
+                //      2. System.Runtime.InteropServices.DllImportResolver callbacks
```

Comment:
Can you please point me to the document describing this order?
I found from [here](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.nativelibrary.setdllimportresolver?view=netcore-3.0#remarks) that the callback resolver is the first attempt to resolve native library loads initiated by this assembly.

Reply:
The comment came from an CoreFX issue that is not reliable source.
Now I found [docs](https://github.com/dotnet/docs/blob/b9d8b8c1af31c3b5e4577ea226d1a02ece32091d/docs/core/dependency-loading/loading-unmanaged.md)

The comment was incorrect. Fixed.

Reply:
Great doc! Thanks!!